netsetup/srv_gatt_wifi.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2019, CESAR. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import dbus
+import struct
+from dbus.service import method as dbus_method
+
+from ble_util.gatt_service import Service
+from ble_util.characteristic import Characteristic
+from ble_util.errors import InvalidValueLengthException
+
+
+class WifiService(Service):
+    WIFI_UUID = "a8a9e49c-aa9a-d441-9bec-817bb4900e40"
+
+    def __init__(self, bus, index):
+        Service.__init__(self, bus, index, self.WIFI_UUID, True)
+        self.add_characteristic(SSIDCharacteristic(bus, 0, self))
+        self.add_characteristic(PSWDCharacteristic(bus, 1, self))
+
+
+class SSIDCharacteristic(Characteristic):
+    SSID_UUID = "a8a9e49c-aa9a-d441-9bec-817bb4900d41"
+
+    def __init__(self, bus, index, service):
+        Characteristic.__init__(self, bus, index, self.SSID_UUID,
+                                ["read", "write"], service)
+        self.value = []
+
+    def __convert_array_to_str(self, array):
+        return ''.join(chr(i) for i in array)
+
+    def ReadValue(self, options):
+        logging.info('WiFi SSID: %s' % self.__convert_array_to_str(self.value))
+        return self.value
+
+    def WriteValue(self, value, options):
+        if not value:
+            logging.error('Invalid value length')
+            raise InvalidValueLengthException()
+        logging.info('WiFi SSID: %s' % self.__convert_array_to_str(value))
+        self.value = value
+
+
+class PSWDCharacteristic(Characteristic):
+    PSWD_UUID = "a8a9e49c-aa9a-d441-9bec-817bb4900d42"
+
+    def __init__(self, bus, index, service):
+        Characteristic.__init__(self, bus, index, self.PSWD_UUID,
+                                ["encrypt-write"], service)
```

Comment:
Use "write" only.
Currently, the knot app setup is not doing any security process.